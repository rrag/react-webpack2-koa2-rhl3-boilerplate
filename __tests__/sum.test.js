import React from "react"
import renderer from "react-test-renderer";

import Layout from "../src/Layout"

it("renders foobar", () => {
	const tree = renderer.create(
		<Layout>foobar</Layout>
	).toJSON();
	expect(tree).toMatchSnapshot();
});
