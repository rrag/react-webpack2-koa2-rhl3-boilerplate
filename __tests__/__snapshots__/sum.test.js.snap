exports[`test renders foobar 1`] = `
<div>
  <h1>
    Hello, world!
  </h1>
  foobar
</div>
`;
